index.html

```diff
@@ -17,7 +17,7 @@
     <section>
         <h1 class="header">Hey there.
             <br>I’m Ian.
-           <hx class="text_color">I’m a software developer</hx>
+           <h1 class="text_color">I’m a software developer</h1>
```

Comment:
`HTML` header tags follow a hierarchy, from `<h1>` to `<h6>`.

- `H1` tags are used to denote the most important text, such as the main theme or title of a content.
- `H2` and `H3` tags are commonly used as subheadings.
- Finally, `H4`, `H5`, and `H6` tags may be used to provide further structure within those subsections.

-  [ ] As suggested by the previous reviewer please make sure that your subtitle (I’m a software developer) uses a header tag `h2` . This way will be semantically correct and the screen readers and other technologies will thank you 😄.

Reply:
@shella12 I have updated though I have finished review requests